public/app/features/alerting/unified/utils/notification-policies.ts

```diff
@@ -152,7 +152,7 @@ function findMatchingAlertGroups(
 
 export type InhertitableProperties = Pick<
   Route,
-  'receiver' | 'group_by' | 'group_wait' | 'group_interval' | 'repeat_interval' | 'mute_time_intervals'
+  'receiver' | 'group_by' | 'group_wait' | 'group_interval' | 'repeat_interval'
```

Comment:
Maybe we could reuse a list to avoid this duplication of the list of inherited properties (in here  ?: 
```

const INHERITABLE_PROPERTIES = ['receiver', 'group_by', 'group_wait', 'group_interval', 'repeat_interval'] as const;

export type InhertitableProperties = Pick<Route, typeof INHERITABLE_PROPERTIES[number]>;

// inherited properties are config properties that exist on the parent route (or its inherited properties) but not on the child route
function getInheritedProperties(
  parentRoute: Route,
  childRoute: Route,
  propertiesParentInherited?: Partial<InhertitableProperties>
) {
  const fullParentProperties = merge({}, parentRoute, propertiesParentInherited);

  const inheritableProperties: InhertitableProperties = pick(fullParentProperties, INHERITABLE_PROPERTIES);

```

Reply:
Great suggestion – will add